openapi/kas-fleet-manager.yaml

```diff
@@ -1,9 +1,9 @@
 openapi: 3.0.0 # need this as first line to allow some IDEs to know this is an openapi document.
 # All endpoints defined here will be discoverable by all users. If there is a need to keep some endpoints "private", add them to the "managed-services-api-private.yaml" file
 info:
-  title: Kafka Service Fleet Manager
```

Comment:
The name  `Kafka Service Fleet Manager` was a suggested name back then and this is where `kas-fleet-manager` came from. Has this been changed?

Reply:
Yes. I agree. This is suggested name for "project name". Internal architecture and all internal communication.
For our customers and users however that might be quite cryptic term etc.